test/client.js

```diff
@@ -3564,6 +3564,32 @@ describe('client API', function() {
         importedClient.isPrivKeyEncrypted().should.be.true;
       });
 
+      it('should export & import decrypted when password is supplied', function() {
+        clients[0].encryptPrivateKey('password');
+
+        var exported = clients[0].export({
+          password: 'password'
+        });
+
+        var err;
```

Comment:
This section could be on a different test case
"should throw err is password is wrong"